app/views/declaration/items_summary.scala.html

```diff
@@ -30,79 +42,142 @@
-                    <th scope="col">@messages("supplementary.itemsSummary.noOfPackages")</th>
-                    <th scope="col" colspan="2"></th>
-                </tr>
-            </thead>
-            <tbody>
-
-            @items.zipWithIndex.map { case (item, index) =>
-                <tr id="item_@index">
-                    <th id="item_@index--sequence_id" scope="row">@item.sequenceId</th>
-                    <td id="item_@index--procedure_code">@item.procedureCodes.map(_.procedureCode).getOrElse("") </td>
-                    <td id="item_@index--item_type" >@item.commodityDetails.map(_.combinedNomenclatureCode) </td>
-                    <td id="item_@index--package_count">
-                        <ol type="1">@item.packageInformation.getOrElse(List.empty).map(packageInfo =>
-                            packageInfo.numberOfPackages.map(packages => <li>{packages}</li>)
-                        )
-                        </ol>
-                    </td>
-                    <td>
-                        <a id="item_@index--change" href="@routes.ProcedureCodesController.displayPage(mode, item.id)">
-                            <span aria-hidden="true">@messages("site.change")</span>
-                            <span class="visuallyhidden">@{messages("supplementary.itemsAdd.change.hint", item.sequenceId)}</span>
-                        </a>
-                    </td>
-                    <td>
-                        <a id="item_@index--remove" class="button button--gray" href="@routes.ItemsSummaryController.removeItem(mode, item.id)">
-                            <span aria-hidden="true">@messages("site.remove")</span>
-                            <span class="visuallyhidden">@{messages("supplementary.itemsAdd.remove.hint", item.sequenceId)}</span>
-                        </a>
-                    </td>
-                </tr>
-            }
-
-            </tbody>
-        </table>
+        @saveAndContinue("site.save_and_continue")
     }
-    @helper.form(routes.ItemsSummaryController.submit(mode), 'autocomplete -> "off") {
-        @helper.CSRF.formField
-
-        <div class="section">
-            @if(items.isEmpty) {
-                @components.buttons.add_button("site.add.item", "site.add.item")
-            } else {
-                @components.buttons.add_button("site.add.anotherItem", "site.add.anotherItem")
-            }
+    <p class="govuk-body">
+        @saveAsDraft("site.save_and_come_back_later")
+    </p>
+}
+
+@changeLink(item: ExportItem) = {
+    <a class="govuk-link" href="@routes.ProcedureCodesController.displayPage(mode, item.id).url">
+        @messages("site.change")
+        <span class="govuk-visually-hidden">
+            @messages("supplementary.itemsAdd.change.hint", item.sequenceId)
+        </span>
+    </a>
+}
 
-            @if(items.nonEmpty) {
-                @components.buttons.save_and_continue_button()
+@removeButton(item: ExportItem) = @{
```

Comment:
Same comment as "addButton"

Reply:
I have refactored the code and have reused it.